drkafka/config/doctorkafka.properties

```diff
@@ -42,54 +61,125 @@ doctorkafka.brokerstats.consumer.ssl.truststore.location=truststore_path
 doctorkafka.brokerstats.consumer.ssl.truststore.password=truststore_password
 doctorkafka.brokerstats.consumer.ssl.truststore.type=JKS
 
+## default enabled monitors, in evaluation order
+doctorkafka.enabled_monitors=interval_timer,maintenance_mode,no_stats_broker_monitor,dead_broker_monitor,urp_monitor
```

Comment:
what are these monitors for?  it will be confusing for users that uses doctorkafka for the first time.  this will also give users more chances to make mistakes.

if we are going through the plug-in direction. shall we have a set of monitors that enables by default?  will doctorkafka work properly if users remove a few `monitors` or `actions` from the list?